templates/index.html

```diff
@@ -290,34 +291,38 @@ <h1>Web &amp; design team deployment portal</h1>
       let release = domainRow.querySelector('.release');
 
       if ('error' in domainJson) {
-        imageTag.setAttribute('colspan', '3');
-        imageTag.innerHTML = `<small style="color: red; font-style: italic">${domainJson.error}</small>`;
-        commit.remove();
-        humanTime.remove();
+          imageTag.setAttribute('colspan', '3');
+          imageTag.innerHTML = `<small style="color: red; font-style: italic">${domainJson.error}</small>`;
+          commit.remove();
+          humanTime.remove();
       } else {
-        imageTag.innerHTML = `<code>${domainJson['imageTag']}</code>`;
-        commit.innerHTML = `<a href="https://github.com/${domainRepositories[domainJson['domain']]}/commit/${domainJson['commitId']}">${domainJson['commitId']}</a>`;
-        humanTime.textContent = domainJson['humanTime'];
-        humanTime.dataset.sort = domainJson['epochTime'];
-        if (release) {
-          release.innerHTML = `<a style="width: 100%" class="p-button--neutral" href="https://jenkins.canonical.com/webteam/securityRealm/commenceLogin?from=%2Fwebteam%2Fjob%2F${domainJson['domain']}-production%2Fparambuild%2F%3Fimage_tag%3D${domainJson['imageTag']}">` +
-            `  Release` +
-            `</a>`;
+          let stagingCommitID = domainJson['staging']['commitId'];
+          let productionCommitID = domainJson['production']['commitId'];
+          imageTag.innerHTML = `<code>${domainJson['staging']['imageTag']}</code>`;
+          commit.innerHTML = `<a href="https://github.com/${domainRepositories[domainJson['staging']['domain']]}/commit/${stagingCommitID}">${stagingCommitID}</a>`;
+          humanTime.textContent = domainJson['staging']['humanTime'];
+        humanTime.dataset.sort = domainJson['staging']['epochTime'];
+          if (release) {
+            if (stagingCommitID !== productionCommitID) {
+              release.innerHTML = `<a style="width: 100%" class="p-button--neutral" href="https://jenkins.canonical.com/webteam/securityRealm/commenceLogin?from=%2Fwebteam%2Fjob%2Fdeploy-to-${domainJson['staging']['domain']}%2Fparambuild%2F%3Fimage_tag%3D${domainJson['staging']['imageTag']}">` +
+                                  `  Release 🚀` +
+                                  `</a>`;
+            } else {
+              release.innerHTML = '<a style="width: 100%" class="p-button--neutral" href="#" disable="disable">Up to date 👍</a>';
```

Comment:
Couldn't you make this a `<button disabled="true">`? 'Cos a link to `#` is a somewhat misleading experience.

Reply:
Good spot, forgot I wasn't finished there. Done